src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,36 @@
+package duke;
```

Comment:
Great that you classified your packages differently, and similar for the other classes in `Duke.exception`, etc